src/crypto/algorithms/megolm.js

```diff
@@ -1370,10 +1400,14 @@ MegolmDecryption.prototype.onRoomKeyEvent = function(event) {
         keysClaimed = event.getKeysClaimed();
     }
 
+    const extraSessionData = {};
+    if (content["org.matrix.msc3061.shared_history"]) {
+        extraSessionData.sharedHistory = true;
```

Comment:
Should this locally stored session data be prefixed with the MSC while it's an experiment, or do we assume it's not necessary to guard local data like that?

Reply:
In general, local data doesn't need to be guarded in the same way as data sent between client, as the local client can usually manage any changes itself.